src/lwc/geListView/geListView.js

```diff
@@ -483,16 +524,16 @@ export default class geListView extends LightningElement {
         }
 
         // Turn fields in the 'Name' column into URLs
-        if (fieldApiName === NAME) {
-            columnEntry.type = URL;
-            columnEntry.fieldName = URL;
+        if (columnEntry.type === URL) {
+            columnEntry.fieldName = fieldApiName + '_' + URL;
             columnEntry.typeAttributes = {
                 label: {
                     fieldName: fieldApiName
                 },
                 target: _SELF
```

Comment:
What do you think about adding a public property 'target' and applying it here if provided otherwise fallback to `_SELF`?

Reply:
yeah that sounds good to me. done!